### PR TITLE
Docs/unencrypted checkins

### PR DESCRIPTION
**Pull request by FelixRottler**
- small improvement to clarify the difference between distribution app feature and submission env variable regarding unencrypted checkins

### CODE REVIEW

File: docs/DISTRIBUTION.md [hunk 1]
```diff
@@ -344,3 +345,18 @@ The content part of the body is also checked to be a valid X509 certificate.
...
+```yaml
+    app-features:
+      - label: unencrypted-checkins-enabled
+        value: ${EVREG_UNENCRYPTED_CHECKINS_ENABLED:0}
+```
+
+#### Additional Note
+
+**Not to be confused** with [```UNENCRYPTED_CHECKINS_ENABLED```](../services/submission/src/main/resources/application.yaml) from the **submission service**, which indicates whether
+submission still accepts default check-ins or only accepts encrypted check-ins and is of type `Boolean`.
```
Comment (Ein-Tim) [inline docs/DISTRIBUTION.md:362]: Line break too much?

File: docs/SUBMISSION.md [hunk 2]
```diff
@@ -153,3 +181,17 @@ From a distribution service perspective, generated keys are indistinguishable fr
...
+This flag is used to control whether check-ins ( which is the current default) are still allowed or not accepted anymore.
+
+```yaml
+unencrypted-checkins-enabled: <true or false>
+```
+
+### Additional Note
+
+**Not to be confused** with [```EVREG_UNENCRYPTED_CHECKINS_ENABLED```](../services/distribution/src/main/resources/application.yaml) from the distribution service
+which indicates for the mobile clients which feature is enabled and is of type ```Integer```.
```
Comment (Ein-Tim) [inline docs/SUBMISSION.md:197]: Line break too much?

File: docs/SUBMISSION.md [hunk 3]
```diff
@@ -27,7 +36,19 @@ The payload to be sent by the mobile applications is defined in the [submission_
...
         [default = 144]; // defaults to 24 hours
-}
+    // Type of diagnosis associated with a key.
+    optional ReportType report_type = 5 [default = CONFIRMED_TEST];
+    // Number of days elapsed between symptom onset and the TEK being used.
+    // E.g. 2 means TEK is 2 days after onset of symptoms.
+    optional sint32 days_since_onset_of_symptoms = 6;
+  }
+
+  message CheckIn {
```
[the remark this answers is not in the document]
Comment (FelixRottler) [reply docs/SUBMISSION.md:46]: are missing, thank you ( this was before the merge)

File: docs/SUBMISSION.md [hunk 4]
```diff
@@ -37,6 +58,13 @@ CWA-Authorization: TAN <TAN>
 CWA-Fake: <0 or 1>
 ```
 
+The response headers returned by this endpoint include the number if check-ins that were successfully saved and the number of check-ins that were filtered out before further processing.
```
Comment (roesslerj) [inline docs/SUBMISSION.md:69]: ... the number **of** check-ins ... sorry to be nit-picking here.

**Review by roesslerj — CHANGES_REQUESTED** [review]

**Review by AliceBilc — APPROVED** [review]

**Review by roesslerj — APPROVED** [review]

**Pull request merged**